### DISCUSSION TITLE
Explore feed is not showing starred repos updates anymore

### DISCUSSION THREAD

**Discussion by xximj** [community/community#145161] [Other Feature Feedback, Questions, & Ideas]
I used to track my starred repos updates and releases using the explore tab on iOS app but this feature is no longer available wtf.

**Comment by aspeckofdustin** [comment]
Resharing my comment from another related issue [here](https://github.com/orgs/community/discussions/145214#discussioncomment-11598958), since it's related:

> 👋 Hi there, Dustin from the Sunset product team here! Our team works on an adjacent product area but because the Feed is highly trafficked and used; although we aren't the Feed team per se, we have been working internally to see if there are parts of the logic or infrastructure that we can fix to help you all have a better experience.
> 
> You should be able to see the content from new releases so that is a major bug that I will flag back with the team. If I get any updates and we can push a fix to make sure that you're all seeing the right, relevant activity updates, I will circle back here and keep you all posted!
